### DISCUSSION TITLE
Domain transfer out

### DISCUSSION THREAD

**Discussion by wrux** [vercel/community#5579] [Account Issue]
### Question

I triggered an outgoing domain transfer sometime last week. It's not showing anything in Vercel and has still not transferred.

Is there a way to check if transfer lock is enabled or to see if there's a pending outgoing transfer on the domain>

**Comment by joshuafbarker** [comment]
I am trying the same thing... tried it a few weeks ago and it sat in limbo for like 2 weeks and never transferred.. and I never got a confirmation email from vercel either... would really like to get my domain transferred out
